syntax = "proto3";

package drltt_proto;

// State coordinates in fixed space frame
message BodyState {
    // x-coordinate in [m]
    optional float x = 1;
    // y-coordinate in [m]
    optional float y = 2;
    // heading in [rad]
    optional float r = 3;
}
